Scripts/summaryStatistics/biophysics/density_calculation.py

```diff
@@ -203,7 +203,7 @@ def calculate_colony_density(img, fig_export_path=''):
     
```

Comment:
line 92-97 above: should 5 be replaced with add_pixels_to_img_border ?

Reply:
That makes sense.